routes/verifyToken.js

```diff
@@ -0,0 +1,20 @@
+//!!Creating a function that can be added to a router to make it protected.
+import jsonwebtoken from "jsonwebtoken";
```

Comment:
use import jwt from jsonwebtoken to avoid redundant imports

Reply:
All solved all above. Thank you